src/elm/Ui/Button.elm

```diff
@@ -0,0 +1,124 @@
+module Ui.Button exposing
+    ( ButtonMode(..)
+    , ThemeColor(..)
+    , button
+    , coloredIcon
+    , primary
+    , primaryDefault
+    , secondary
+    , secondaryDefault
+    , tertiary
+    )
+
+import Html as H exposing (Html)
+import Html.Attributes as A exposing (disabled)
+import Html.Events as E
+import Html.Extra as Html
```

Comment:
Isn't this a bit confusing? Wouldn't it be better to not use `as Html` and then use `Html.Extra.viewMaybe` further down? Or at least give it a better name like `HtmlExtra` or even `HE`.